TradeItIosTicketSDK2/TradeItTradingTicketViewController.swift

```diff
@@ -242,6 +248,7 @@ class TradeItTradingTicketViewController: TradeItViewController, UITableViewData
         self.order.symbol = selectedSymbol
         self.clearMarketData()
         self.updateMarketData()
+        self.selectedSymbolChanged = true
```

Comment:
maybe we make this `self.selectedSymbolChanged = self.order.symbol != selectedSymbol`? IMO it fits better definition wise.

This controller flow is a little confusing as well -- not sure we can do much about it right now, but it requires knowing a thing or two about when different actions are called